### DISCUSSION TITLE
[Feature Request] Add `slot:number` to the Step component.

### DISCUSSION THREAD

**Discussion by crotanite** [primefaces/community#2858] [PrimeVue]
In the Step component it would be nice to have more control over the number section, a change like:

```vue
<span :class="cx('number')" v-bind="getPTOptions('number')"><slot name="number">{{ activeValue }}</slot></span>
```

**Comment by cagataycivici** [comment]
Thanks, better to add this to the issue tracker of PrimeVue, PRs are always welcome.
